engine/admin/logging/awslogs.md

```diff
@@ -168,7 +168,7 @@ The following `strftime` codes are supported:
 | `%p` | AM or PM.                                                        | AM       |
 | `%M` | Minute as a zero-padded decimal number.                          | 57       |
 | `%S` | Second as a zero-padded decimal number.                          | 04       |
-| `%L` | Milliseconds as a zero-padded decimal number.                    | 123      |
+| `%L` | Millisecond as a zero-padded decimal number.                    | 123      |
```

Comment:
Please revert this one.

Reply:
Whould you please give me some clue why the plural form ("milliseconds") should be used here? Thanks.

Reply:
Because you're almost never going to set it to `1`. It is technically a "number of milliseconds". That would be the most correct way to say it, but it's a bit wordy.

Reply:
Thanks. Then, what about the next item "microsecond" in the table? Should it also use the plural form?

Reply:
Yes, good catch! 👍